packages/block-library/src/navigation-menu/block-colors-selector.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { IconButton, Dropdown, Toolbar } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { DOWN } from '@wordpress/keycodes';
+import { ColorPaletteControl, ContrastChecker } from '@wordpress/block-editor';
+
+/**
+ * Color Selector Icon component.
+ * @return {*} React Icon component.
+ * @constructor
+ */
+const ColorSelectorIcon = ( { style } ) =>
+	<div className="block-editor-block-colors-selector__icon-container">
+		<div
+			className="colors-selector__state-selection wp-block-navigation-menu-item"
+			style={ style }
+		>
+			{ __( 'Aa' ) }
```

Comment:
We should probably use `_x()` here to allow us to pass a description for the translator. `Aa` on itself doesn't provide sufficient context.

Reply:
This would also be better as an icon.

Reply:
Maybe we should just simply remove the i18n here. cc @shaunandrews.

Reply:
If we look at the H1, H2, … icons in the heading, they are combination of icon for the "H" part and text for the number. We can probably use either of those. If we go with the text, I'd remove the i18n as you suggest.